packages/sanes-chrome-extension/src/extension/background/actions/createPersona/requestSenderWhitelist/index.unit.spec.ts

```diff
@@ -19,9 +17,7 @@ describe('SenderWhitelist', () => {
     SenderWhitelist.block(SENDER_TWO);
 
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    const numBlocked = localStorage.getItem(BLOCKED_URLS_KEY)!.split(SEPARATOR).length;
-
-    expect(numBlocked).toBe(2);
+    expect(SenderWhitelist['blocked']!.length).toBe(2);
```

Comment:
You can call me old, but accessing a private static field in this way (first of all, I did not know that was possible) makes me feel strange.

But I understand your point, and seeing this is a test I would add this comment:
```
// SenderWhitelist does not expose the array of blocked domains, so this is an exceptional
// use case which can not be used in regular code because it violates private modifier of blocked variable.
```

Reply:
Sure, I believe it would be better to have a getBlocked() method, but I didn't want to extend the API only for the sake of tests.